packages/hint-no-disallowed-headers/src/_locales/en/messages.json

```diff
@@ -11,6 +11,34 @@
         "description": "Report message when the 'server' header contains more than the server name",
         "message": "The 'server' header should only contain the server name."
     },
+    "disallowedExpiresHeader": {
+        "description": "Report message when the response includes the Expires header",
+        "message": "The 'Expires' header should not be used, 'Cache-Control' should be preferred."
+    },
+    "disallowedHostHeader": {
+        "description": "Report message when the response includes the Host header",
+        "message": "The 'Host' header should not be used, it is a request header only."
+    },
+    "disallowedP3PHeader": {
+        "description": "Report message when the response includes the P3P header",
+        "message": "The 'P3P' header should not be used, it is a non-standard header only implemented in Internet Explorer."
+    },
+    "disallowedPragmaHeader": {
+        "description": "Report message when the response includes the Pragma header",
+        "message": "The 'Pragma' header should not be used, it is deprecated and is a request header only."
+    },
+    "disallowedViaHeader": {
+        "description": "Report message when the response includes the Via header",
+        "message": "The 'Via' header should not be used, it is a request header only."
+    },
+    "disallowedXFrameOptionsHeader": {
+        "description": "Report message when the response includes the X-Frame-Options header",
+        "message": "The 'X-Frame-Options' header should not be used, the same effect can be achieved with more consistent support using the 'Content-Security-Policy' header."
+    },
+    "disallowedXUACompatibleHeader": {
```

Comment:
Apologies for missing this in our earlier discussion, but `X-UA-Compatible` overlaps with our recommendations from `hint-highest-available-document-mode` (https://webhint.io/docs/user-guide/hints/hint-highest-available-document-mode/). 

That hint is a touch more nuanced: it recommends removing the meta tag or header if targeting IE11+, but recommends including it if for some crazy reason you're still targeting IE10 or below to ensure you're running in the latest standards mode.

We should probably not include `X-UA-Compatible` as part of this hint to avoid redundant reports.

Reply:
Good to know!
I changed the code to not include `X-UA-Compatible`.